### ISSUE TITLE
Using no convolution better than DCRNN paper result?

### ISSUE THREAD

**Issue by Noahprog** [chnsh/DCRNN_PyTorch#7]
Hi @chnsh,

Hope you're doing well. While testing your pytorch implementation of the DCRNN code, I stumbled across a weird result. When turning off convolution with `max_diffusion_step=0` the result was greatly improved over the original dcrnn paper result.

I tested this on the _PeMS dataset_, configurations are: 
<img width="417" alt="Screenshot 2020-06-03 at 16 35 01" src="https://user-images.githubusercontent.com/31852504/83650096-3c275e80-a5b8-11ea-9075-aeed7cb80810.png">
link to file: [model_config_issue.txt](https://github.com/chnsh/DCRNN_PyTorch/files/4723023/model_config_issue.txt). In short, I simplified the model a bit by doing the _15min forecast_, using _'laplacian' filter_, _no curriculum learning_ and only _60 epochs_. Of course also _`max_diffusion_step=0`_ is used to discard using a convolution.

This resulted in `val_mae: 1.2388` at 60th epoch as can be seen in the snippet or full [info.log](https://github.com/chnsh/DCRNN_PyTorch/files/4723047/info.log). <img width="953" alt="Screenshot 2020-06-03 at 16 29 53" src="https://user-images.githubusercontent.com/31852504/83649506-907e0e80-a5b7-11ea-92a4-e14d03f17da7.png"> This result is better than the full blown published DCRNN which reported `val_mae: 1.38`. The fact that even a simpler model without convolution is better than the original DCRNN should raise concern about the soundness of this implementation. This is kind of the same problem as issue #3. 

I'm not familiar with tensorflow, that is why your implementation has given me much help with my thesis. Because this observation could probably bottleneck me down the road I would like to pin down the reason for this behaviour as early as possible. I think you have more insights in the workings of the original tensorflow implementation of DCRNN, thus I would like to ask you to have another look in finding the problem. I have a gut feeling the problem lies somewhere in the calculation of the error/loss.

Hope you can find the time to look into this issue. Thanks in advance.

**Comment by Noahprog** [comment]
For the answer of this issue I would like to refer to issue #3
